mplex/src/codec.rs

```diff
@@ -49,6 +49,14 @@ impl Elem {
             Elem::Reset { substream_id, .. } => substream_id,
         }
     }
+
+    pub fn is_open(&self) -> bool {
```

Comment:
Nit: would prefer `is_open_msg()` so that we know we're talking about an open message and not an open substream.